flexmeasures/api/v3_0/tests/test_sensors_api.py

```diff
@@ -28,7 +28,6 @@ def test_fetch_one_sensor(
     assert response.status_code == 200
     assert response.json["name"] == "some gas sensor"
     assert response.json["unit"] == "m³/h"
-    assert response.json["generic_asset_id"] == 4
```

Comment:
Any idea why this is not returned anymore?

Reply:
Great question! The asset id is not explicitly assigned anywhere but is created when the asset is created. This means it's subject to change when changes are made to conftest. Because of this the test could break with future changes as well and changing the number in this test would not actually make this test valuable. 

I've changed the test to first get the generic_asset linked to this sensor and check that it's linked to the "incineration line" asset.